csharp/samples/getting-started-receiving.md

```diff
@@ -0,0 +1,201 @@
+# Get started with Event Hubs in .NET
+
+## What will be accomplished
+
+In this tutorial, we will write a console application to receive messages from an Event Hub using the **EventProcessorHost**.
+
+## Prerequisites
```

Comment:
But I guess for the sake of this example it is assumed to be .NET Core. Good call.